sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/GpuBroadcastNestedLoopJoinExec.scala

```diff
@@ -637,3 +646,25 @@ case class GpuBroadcastNestedLoopJoinExec(
     }
   }
 }
+
+class ConditionalExistenceJoinIterator(
```

Comment:
Similarly this is not the iterator to use for any conditional existence join but specifically a nested loop join.  I suggest calling this `ConditionalNestedLoopExistenceJoinIterator` or something similar.

Reply:
done